content/en/docs/concepts/policy/pid-limiting.md

```diff
@@ -0,0 +1,43 @@
+---
+reviewers:
+- derekwaynecarr
+title: PID Limiting
+content_type: concept
+weight: 10
```

Comment:
I recommend setting a higher value, as there are other documents in the policy section that should come first.

Reply:
I should have read contributing.md. Just copied from other file. Let me take a look

Reply:
this is the doc I found: https://v1-16.docs.kubernetes.io/docs/contribute/style/content-organization/#page-order I updated weight for more articles. Was it the right decision? Or just assign 40 to the new page?